.github/workflows/r-valgrind.yaml

```diff
@@ -14,20 +14,45 @@ jobs:
       image: rocker/r2u:latest
     steps:
       - uses: actions/checkout@v3
+
       - name: SessionInfo
         run: R -q -e 'sessionInfo()'
+
       - name: System Dependencies
         run: apt update -qq && apt upgrade --yes && apt install --yes --no-install-recommends valgrind cmake git
+
+      - name: Set additional repositories (Linux)
+        run: |
+          rversion <- paste(strsplit(as.character(getRversion()), split = '\\.')[[1L]][1:2], collapse = '.')
+          codename <-  system('. /etc/os-release; echo ${VERSION_CODENAME}', intern = TRUE)
+          repo <- "https://tiledb-inc.r-universe.dev"
+          (opt <- sprintf('options(repos = c("%s/bin/linux/%s/%s", "%s", getOption("repos")), timeout = 300L)', repo, codename, rversion, repo))
+          print("RVERSION")
+          print(rversion)
+          print("CODENAME")
+          print(codename)
+          print("REPO")
+          print(repo)
```

Comment:
Can you please remove lines 30 to 35?  

Line 29 does not need the outer `(  )` as there is not printing here.

Reply:
The outer parentheses on line 29 show up in CI logs as
![image](https://github.com/single-cell-data/TileDB-SOMA/assets/11035068/8a48ff11-edcc-4ffd-9587-290955b91e1f)

Reply:
And ... ?

If you want to print, be explicit. Maybe use ':::' groupings.  Obscure code helps noone.

Reply:
This was debug; I'll remove